dbt/include/redshift/macros/relations/drop.sql

```diff
@@ -0,0 +1,7 @@
+{% macro redshift__get_drop_relation_sql(relation) %}
+    {%- if relation.is_materialized_view -%}
+        {{ redshift__drop_materialized_view(relation) }}
+    {%- else -%}
+        drop {{ relation.type }} if exists {{ relation }} cascade
+    {%- endif -%}
+{% endmacro %}
```

Comment:
when these changes roll out to other materializations, we'll likely see more macros called within here.. right?

Reply:
Ideally this gets standardized into `dbt-core`. I'll look back at the git blame, but I believe this was a renamed version of drop_relation that I then added materialized view to as an option. But we really should audit where this used and use the version that's in `dbt-core`.

In general, yes, we will see more relations (so table and view) show up in these "switchboard" macros (e.g. `get_drop_sql`) and more calls to them as well, with less calls to relation-specific macros (e.g. `get_drop_table_sql`). For this MV work I am trying to touch as few macros/files as possible as the sheer number of files affected in the last approach caused heartburn. At a bare minimum, I need all operations for MVs and drop/rename for all relation types (so I can replace tables and views with MVs and vice versa).